packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx

```diff
@@ -85,9 +197,19 @@ const TableCalculationModal: FC<Props> = ({
         defaultValues: {
             name: tableCalculation?.displayName,
             sql: tableCalculation?.sql,
+            format: {
```

Comment:
I started migrating react form to mantine use-form, but  because one of the components is an ace-editor I think is not that simple, so didn't refactor much, and new componets are all mantine.